netbox/dcim/models/device_components.py

```diff
@@ -87,48 +87,43 @@ class CableTermination(models.Model):
     class Meta:
         abstract = True
 
-    def trace(self, position=1, follow_circuits=False, cable_history=None):
+    def get_peer_port(self):
```

Comment:
I'm on the fence about this, but I'm leaning toward _not_ declaring this on all device components. IMO it could be confusing to have a `get_peer_port()` method on a model like ConsolePort, which can never have a peer port (though people might understandably expect it to return a ConsoleServerPort). I do appreciate that it makes for cleaner calls, though.

Reply:
Rename it to `comes_out_again_on_port()`? :)

Reply:
The main reason I added it in the superclass is to cover situations where people connect things together that we currently don't expect. I am sure reality is weirder than I can imagine… With this in the base class the trace method will not malfunction but just stop.

Actually: we want to be able to trace console connections. They can be patched through patch panels as well, so a trace is valuable. I strongly feel we should keep the method but possible rename it.